python/server/ServerWrapper.cpp

```diff
@@ -321,6 +321,18 @@ namespace {
         return py_items;
     }
 
+    // Wrapper for starting buildings
+    list LoadStartingBuildings() {
+        list py_items;
+        std::vector<ItemSpec> buildings;
+        parse::starting_buildings(buildings);
+        for (std::vector<ItemSpec>::const_iterator building = buildings.begin()
+                 ; building != buildings.end(); ++building) {
```

Comment:
weird indenting, move ; to end of line, move { to new line for a multi-line for block